compute_endpoint/globus_compute_endpoint/providers/kubernetes/kube.py

```diff
@@ -288,7 +289,10 @@ def _create_pod(
             )
 
         # Create the enviornment variables and command to initiate IPP
-        environment_vars = client.V1EnvVar(name="TEST", value="SOME DATA")
+        env = []
+        for var_name in ("GC_TASK_TIMEOUT",):
```

Comment:
Is the idea that we'll only pass along env vars that we explicitly specify?

Reply:
Correct.

Reply:
But I think it's all it needs.  At least, that's the thinking per the commit message:

> ... If passing information from the parent pod (the endpoint) to the child pods (the workers) becomes an issue in the future, the intended approach is to "deal with it then" in a more complete way. For now, then, though not ideal, this commit is "good enough."

And, recall that these are the _injected_ (and by "these," I mean "this") variables, not the only environment variables for the kube process.